src/background/network.js

```diff
@@ -398,6 +398,8 @@ export class Network extends Component {
     // eslint-disable-next-line verify-await/check
     browser.experiments.proxyutils.FTPEnabled.set({value: false});
     // eslint-disable-next-line verify-await/check
+    browser.proxy.settings.set({value: {respectBeConservative: false}});
```

Comment:
This will overwrite all other proxy settings as well, just wanting to make sure that is the intention.

Reply:
We don't allow custom proxy settings when secure-proxy is active. So yes, it's intentional.